pkg/controller/config/config_controller.go

```diff
@@ -43,7 +43,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/source"
 )
 
-// TODO write a reconciliation process that looks at the state of the cluster to make sure
+// TODO: write a reconciliation process that looks at the state of the cluster to make sure
```

Comment:
this TODO can be removed, we no longer set finalizers

Reply:
done